packages/csv-parser-discount-code/readme.md

```diff
@@ -0,0 +1,11 @@
+# commercetools-csv-parser-discount-code
+
+Convert [commercetools discount codes](http://dev.commercetools.com/http-api-projects-discountCodes.html) CSV data to JSON
+
+https://commercetools.github.io/nodejs/cli/csv-parser-discount-code.html
+
+## Install
+
+```bash
+npm install --save @commercetools/csv-parser-discount-code
```

Comment:
I lean more towards adding a `--global` flag, this is a CLI flag so makes sense to add `--global` but no hard opinion on this, I am fine with as is